tests/src/dir.common.props

```diff
@@ -32,6 +32,14 @@
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
     <DefineConstants>$(DefineConstants);DEBUG;TRACE;XUNIT_PERF</DefineConstants>
   </PropertyGroup>
+  
```

Comment:
@danmosemsft could you please review this part?

Reply:
`$(Platform)` can be set in project files, eg `   <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>` although I'm not sure whether your projects actually do. Those are "below" this location (see /pp output)
Not sure whether `$(BuildArch)` is better.
I think if this works -- fine!

Reply:
Ok, I do not know how it works, but we see that ci works fine and I checked that local builds work good too (you can do msbuild /p:Configuration=Debug /p:Platform=(choose what you want) /p:CLRTestPriorityToBuild=1   DevDiv_278523.ilproj and it will determinate the include file correctly)